resource/get_factory.cpp.em

```diff
@@ -18,8 +18,8 @@
 @{
 from ros1_bridge import camel_case_to_lower_case_underscore
 }@
-#include "ros1_bridge/factory.hpp"
 #include "ros1_bridge/builtin_interfaces_factories.hpp"
+#include "ros1_bridge/factory.hpp"
```

Comment:
I would rather revert this change. Before it was including first the factory interface and then all specialized factories. That made sense to me. Also the include order isn't being checked at the moment since some includes are hard coded and some other are being generated. So the overall order won't pass the inter anyway (without custom logic ordering the superset).

Reply:
Feel free to revert it, I wanted to save an iteration but apparently I should have just commented to get feedback rather than commiting the change

Reply:
Np, better being proactive - I appreciate it!